lib/web.ts

```diff
@@ -16,8 +16,14 @@ export default class AutoConsent {
   protected sendContentMessage: MessageSender;
 
   constructor(sendContentMessage: MessageSender, config: Config = null, declarativeRules: RuleBundle = null) {
-    evalState.sendContentMessage = sendContentMessage;
-    this.sendContentMessage = sendContentMessage;
+
+    let scm = sendContentMessage;
+
+    if (enableLogs)
+      scm = async (msg) => {console.log(`Sending content message of type ${msg.type}:`, msg);return sendContentMessage(msg);}
+
+    evalState.sendContentMessage = scm;
+    this.sendContentMessage = scm;
```

Comment:
This refactor is just to add content message debugging, right? I thought we're already logging this on both sides?

Reply:
Yes exactly. I didn't see that in the logs so I added it. I think we log most actions but not every invocation of `sendContentMessage`. I just wanted to see when exactly the app gets the message, we can also remove this again if it's redundant.